src/sentry/static/sentry/app/components/alert.tsx

```diff
@@ -54,43 +81,37 @@ const alertStyles = ({theme, type, system, alignTop}) => css`
   ${system && getSystemAlertColorStyles(theme.alert[type])};
 `;
 
-const AlertWrapper = styled('div', {shouldForwardProp: isPropValid})`
-  ${alertStyles}
-`;
-
 const StyledTextBlock = styled(TextBlock)`
   line-height: 1.4;
   margin-bottom: 0;
   flex: 1;
   align-self: center;
 `;
 
-const Alert = ({type, icon, iconSize, children, className, ...props}) => {
-  let refClass;
-
-  if (type) {
-    refClass = 'ref-' + type;
-  }
-
-  return (
-    <AlertWrapper type={type} {...props} className={cx(refClass, className)}>
+const Alert = styled(
+  ({
+    type = DEFAULT_TYPE,
+    icon,
+    iconSize = '24px',
```

Comment:
Typescript recommends using default parameters instead of defaultProps.

Reply:
Do those default values end up in the emotion css template string? I have used defaultProps a few times on styled components because they were not function components.

Reply:
Okay converted back to defaultProps and just used `!` on iconSize.